lib/auth/auth.go

```diff
@@ -4837,6 +4837,16 @@ func (a *Server) CreateAccessRequestV2(ctx context.Context, req types.AccessRequ
 	if _, err := a.Services.CreateAccessRequestV2(ctx, req); err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	var annotations *apievents.Struct
+	if len(req.GetSystemAnnotations()) > 0 {
+		var err error
+		annotations, err = apievents.EncodeMapStrings(req.GetSystemAnnotations())
+		if err != nil {
+			log.WithError(err).Debugf("Failed to encode access request annotations.")
+		}
+	}
```

Comment:
nit: use a local variable, Debug instead of Debugf.

```suggestion
	if sa := req.GetSystemAnnotations(); len(sa) > 0 {
		var err error
		annotations, err = apievents.EncodeMapStrings(sa)
		if err != nil {
			log.WithError(err).Debug("Failed to encode access request annotations")
		}
	}
```